TEAM_STRUCTURES.md

```diff
@@ -78,12 +79,14 @@ The IPFS Project Working Group Community serves as the point of coordination for
 - Provide coaching to Working Groups to develop their own planning and goal-setting.
 - Facilitate communication and alignment accross the organization.
 - Research, develop and build new ways to help async teams coordinate at scale.
-- Recognize needs of the project and develop solutions to meet them. 
+- Recognize needs of the project and develop solutions to meet them.
 
 ### JavaScript IPFS implementation
 
 - **Coordination**: https://github.com/ipfs/pm/blob/master/JS_CORE_DEV_MGMT.md
-- **Captain**: [David Dias](https://github.com/diasdavid/)
+- **Captain**: [Alan Shaw](https://github.com/alanshaw/)
+- **Technical Manager**: N/A
```

Comment:
@alanshaw I hope with this to finally pass the well deserved and recognized baton without dropping a bunch of Management in tasks on top of you ad-aeternum.

Reply:
👍 woooo!